### DISCUSSION TITLE
Unable to setup copilot.nvim

### DISCUSSION THREAD

**Discussion by navxio** [community/community#53862] [Copilot Conversations]
### Select Topic Area

Bug

### Body

relevant part of my neovim config:
`
		use({
			"github/copilot.vim",
			config = function()
				-- copilot keybinding is C-e
				vim.api.nvim_set_keymap(
					"i",
					"<C-e>",
					[[<Cmd>lua require("copilot").accept("\<CR>")<CR>]],
					{ silent = true, expr = true }
				)
				vim.g.copilot_no_tab_map = true
			end,
		})
`
I've completed the setup process successfully, still I get the error:
`Copilot: <Tab> map has been disabled or is claimed by another plugin`

**Comment by iamcjwilliams** [comment]
@navxio Sorry for the trouble here, have you visited here to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot
